extensions/GitExtension/FileExplorerGitIntegration/Models/GitExecute.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using FileExplorerGitIntegration.Helpers;
+using Microsoft.Windows.DevHome.SDK;
+using Serilog;
+
+namespace FileExplorerGitIntegration.Models;
+
+public class GitExecute
+{
+    public static GitCommandRunnerResultInfo ExecuteGitCommand(string gitApplication, string repositoryDirectory, string command, string arguments)
```

Comment:
Both command and arguments seems redundant at this stage and doesn't really add value. Combine into single param.